osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -33,6 +33,11 @@ public class GameplayClockContainer : Container
 
         public readonly BindableBool IsPaused = new BindableBool();
 
+        public override bool PropagateNonPositionalInputSubTree => !BlockGameplayInput;
```

Comment:
did you try doing this in `RulesetInputManager` as I proposed?

Reply:
I tried, it disable the cursor movements too...